src/UI/class-metadata-renderer.php

```diff
@@ -116,13 +116,13 @@ public function render_metadata( string $meta_type ): void {
 			echo '<script type="application/ld+json">' . wp_json_encode( $metadata ) . '</script>';
 		} else {
 			// Assume `meta_type` is `repeated_metas`.
-			$parsely_post_type = $this->parsely->convert_jsonld_to_parsely_type( $metadata['@type'] );
+			$parsely_post_type = $this->parsely->convert_jsonld_to_parsely_type( $metadata['@type'] ?? '' );
```

Comment:
All attributes of metadata are defined as optional in the interface there we need these extra values in case of `null`. Same reason for other places where we are using `??` with metadata.